src/dpl/src/Place.cpp

```diff
@@ -938,6 +869,15 @@ PixelPt Opendp::binSearch(int x, const Cell* cell, int bin_x, int bin_y) const
 
   if (x > bin_x) {
     for (int i = bin_search_width_ - 1; i >= 0; i--) {
+      Point p(bin_x + i, bin_y);
+      p.setX(p.getX() * site_width_);
+      p.setY(p.getY() * row_height);
+      if (cell->region_) {
+        if (!cell->region_->intersects(p)) {
```

Comment:
no need to nest - just

 if (cell->region_ && !cell->region_->intersects(p)) {